src/main/java/net/sf/jabref/importer/fileformat/RisImporter.java

```diff
@@ -38,56 +38,42 @@
 
     private static final Pattern RECOGNIZED_FORMAT_PATTERN = Pattern.compile("TY  - .*");
 
-    /**
-     * Return the name of this import format.
-     */
     @Override
     public String getFormatName() {
         return "RIS";
     }
 
-    /*
-     *  (non-Javadoc)
-     * @see net.sf.jabref.imports.ImportFormat#getCLIId()
-     */
     @Override
-    public String getCLIId() {
-        return "ris";
+    public List<String> getExtensions() {
```

Comment:
Since I have no idea for which extensions these importers support. 

See above:
> Remark: The methods getExtension and getDescription are not implemented by almost any importer. This should be done. Maybe this is something for the stupro?